atom/browser/atom_browser_client.cc

```diff
@@ -517,10 +524,45 @@ AtomBrowserClient::OverrideSystemLocationProvider() {
 #endif
 }
 
+network::mojom::NetworkContextPtr AtomBrowserClient::CreateNetworkContext(
+    content::BrowserContext* browser_context,
+    bool in_memory,
+    const base::FilePath& relative_partition_path) {
```

Comment:
*_(minor)_* As per style guide, I _think_ in_memory and relative_partition_path should be commented out as unused, e.g.

```cc
content::BrowserContext* browser_context,
bool /*in_memory*/,
const base::FilePath& /*relative_partition_path*/) {
```

Reply:
I believe this is only required when the parameter names are omitted which is not in this case ?

Reply:
I don't understand this comment -- if the name is omitted, then by definition the name can't be commented out? :smiley: 

Unused variables which are obvious from context can have their names omitted entirely, e.g. `class Foo { Foo(Foo&&); };` but those not obvious from context should have a name commented out, e.g. `void Circle::Rotate(double /*radians*/) {}`

[ref](https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions)

Reply:
Hmm seems like I misread the style guide, will update them. Thanks!